qa/tasks/cephfs/test_admin.py

```diff
@@ -91,6 +92,25 @@ def setup_ec_pools(self, n, metadata=True, overwrites=True):
         if overwrites:
             self.run_ceph_cmd('osd', 'pool', 'set', n+"-data", 'allow_ec_overwrites', 'true')
 
+    def wait_till_health_warn(self, health_warn, active_mds_id, sleep=3,
+                              tries=10):
+        errmsg = (f'Expected health warning "{health_warn}" to eventually '
+                  'show up in output of command "ceph health detail". Tried '
+                  f'{tries} times with interval of {sleep} seconds but the '
+                  'health warning didn\'t turn up.')
+
+        with safe_while(sleep=sleep, tries=tries, action=errmsg) as proceed:
+            while proceed():
+                cache_report = self.get_ceph_cmd_stdout(
+                    f'tell mds.{active_mds_id} cache status')
+                log.info(f'cache_report -\n{cache_report}')
+
+                health_report = self.get_ceph_cmd_stdout('health detail')
```

Comment:
JSON isn't needed.

Reply:
I mean it becomes a tad more easier to check for certain health messages.

Reply:
Agreed but JSON is not suitable since the check here too simple and therefore easier when output is treated as string.

Reply:
For this particular case, maybe. But, for programatically validating the presence of a particular string, parsing JSON is immensely helpful. I suggest we change this.

Reply:
Okay, I don't mind doing it but it's redundant to pass `--format json`, loading JSON string into dict, find its exact member that will hold the message, etc. Because `str.find()` does the job with no hassles.

Reply:
Done.

Reply:
Posted.